lib/iris/tests/test_imports.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+
+# import iris tests first so that some things can be initialised before importing anything else
+import iris.tests as tests  # isort:skip
+
+import importlib
+import pathlib
+
+BASE = pathlib.Path("..")
+IGNORE = ["__init__.py", "__pycache__", "pandas.py", "raster.py", "tests"]
+
+
+class TestImports(tests.IrisTest):
```

Comment:
This ran successfully on my local machine, and when I inserted `import dud` into `iris.experimental.ugrid`, the test correctly caught the error as coded below ✔

Reply:
Nice one 👍